### PR TITLE
msx2_cart.xml, msx2_cass.xml, msx2_flop.xml, msx2p_flop.xml: Metadata cleanings

### PR DESCRIPTION
**Pull request by ArcadeShadow**
- Replaced countries' abbreviation by their full name
- Lowercase on descriptive words ("Alt", "Cracked", "Demo", "Remake", ...)

### CODE REVIEW

File: hash/msx2_cart.xml [hunk 1]
```diff
@@ -755,17 +755,17 @@ LZ93A13 (32 pin) - 8KB banks
             <feature name="mapper" value="M60002-0115FP" />
             <feature name="sram" value="D4464C-20L" />
             <dataarea name="rom" size="">
-                <rom name="jr_mate_1_v1.0" size="131072" crc="" sha1="" offset="0" status="nodump" />
-                <rom name="jr_mate_2_v1.0" size="131072" crc="" sha1="" offset="0x20000" status="nodump" />
+                <rom name="jr_mate_1_v1.0" size="131072" crc="" sha1="" offset="0" />
+                <rom name="jr_mate_2_v1.0" size="131072" crc="" sha1="" offset="0x20000" />
                 <!- Kanji rom ->
-                <rom name="tc531000p-6616" size="131072" crc="" sha1="" offset="0x40000" status="nodump" />
+                <rom name="tc531000p-6616" size="131072" crc="" sha1="" offset="0x40000" />
```
Comment (cuavas) [inline hash/msx2_cart.xml:761]: Why did you remove the undumped status flag from these?

File: hash/msx2_cart.xml [hunk 2]
```diff
@@ -842,14 +842,14 @@ LZ93A13 (32 pin) - 8KB banks
             <feature name="slot" value="ascii8" />
             <feature name="mapper" value="NEOS MR6401" />
             <dataarea name="rom" size="131072">
-                <rom name="rp231024d" size="131072" crc="" sha1="" offset="0" status="nodump" />
+                <rom name="rp231024d" size="131072" crc="" sha1="" offset="0" />
```
Comment (cuavas) [inline hash/msx2_cart.xml:845]: This one, too.

**Review by cuavas — COMMENTED** [review]
The diff for msx2_flop.xml is too large to display in GitHub and comment in-place:
* Indentation seems to be messed up for the opening tag of the second `part` element for `cosmoclb`.
* Why did you remove the Japanese titles for `daiva5`, `daiva5a` and `daiva5b`?
* Why did you remove mention that `dslayer4` and `dslayer4a` are apparently pirates of cartridge versions and move them out of the cracks section?
* You missed making “incomplete dump” lowercase for `nvmg0207`.
* You seem to have added a stray space to the disk image label for `pentodys`.
*  Why the extra blank lines before `1942`?  The preceding comment applies to this group of items, it doesn’t need to be separated visually.

**Comment by ArcadeShadow** [comment]
> The diff for msx2_flop.xml is too large to display in GitHub and comment in-place:
> 
>     * Indentation seems to be messed up for the opening tag of the second `part` element for `cosmoclb`.
> 
>     * Why did you remove the Japanese titles for `daiva5`, `daiva5a` and `daiva5b`?
> 
>     * Why did you remove mention that `dslayer4` and `dslayer4a` are apparently pirates of cartridge versions and move them out of the cracks section?
> 
>     * You missed making “incomplete dump” lowercase for `nvmg0207`.
> 
>     * You seem to have added a stray space to the disk image label for `pentodys`.
> 
>     * Why the extra blank lines before `1942`?  The preceding comment applies to this group of items, it doesn’t need to be separated visually.

Undone those changes. I honestly don't remember making these changes. But sorry if my mistake.

File: hash/msx2_cart.xml (hunk 1 above)
Comment (ArcadeShadow) [reply hash/msx2_cart.xml]: > Why did you remove the undumped status flag from these?

Undone those changes.
Same situation as the previous post, I don't remember having made these changes.
But thanks for the warnings!
